app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] I am able to log in but for some reason, I am getting an empty page
check the reason

Reply:
@DeliceLydia 

- Can you give a screenshot of the view in the browser 
- When I log in I am redirected to /users index page

Reply:
@DeliceLydia, When I sign in I am redirected to the /users index page
![image](https://github.com/FaranoshAmini/rails-blog/assets/101713819/03985eff-f9af-4642-a307-c248bdc0c993)